addons/website_event_track/models/event_track.py

```diff
@@ -630,3 +648,91 @@ def _get_track_suggestions(self, restrict_domain=None, limit=None):
         )
 
         return track_candidates[:limit]
+
+    def _get_reminder_times_warning(self):
```

Comment:
:socks: Seems several methods linked to external calendar stuff, could maybe use a common base naming like '_get_track_calendar_description/reminder/...' ? Just proposin' !